Dockerfile

```diff
@@ -108,11 +108,13 @@ ENV RUSTUP_HOME=/usr/local/rustup \
 RUN apk add --no-cache --virtual .build-deps \
         build-base \
         python3-dev \
+        gcc \
+        musl-dev \
     && apk add --no-cache --virtual .fetch-deps \
         curl \
     && apk add --no-cache \
         python3 \
-        bash \
+        bash \ 
```

Comment:
Excessive whitespace please remove

Reply:
Ah, apparently that removal was missing from `.editorconfig`